package.json

```diff
@@ -24,17 +24,13 @@
 
         "mjmcloug.vscode-elixir", 
```

Comment:
This can be removed too

Reply:
sure, it's probably only relevant for very few people in Artsy